templates/server.erb

```diff
@@ -148,14 +148,20 @@ plugin <%= @pam_module_path %> "<%= @pam_module_arguments %>"
 <% if @management -%>
 management <%= @management_ip %> <%= @management_port %>
 <% end -%>
-<% if @up != '' or @down != ''-%>
+<% if (@up and @up != '') or (@down and @down != '') or (@client_connect and @client_connect != '') or (@client_disconnect and @client_disconnect != '') -%>
```

Comment:
the values default to undef and cannot be an empty string. you don't need to check for `!= ''`.